webapp/channels/src/components/widgets/tag/tag.tsx

```diff
@@ -9,7 +9,7 @@ import styled, {css} from 'styled-components';
 import glyphMap from '@mattermost/compass-icons/components';
 import type {IconGlyphTypes} from '@mattermost/compass-icons/IconGlyphs';
 
-export type TagVariant = 'info' | 'success' | 'warning' | 'danger' | 'dangerDim';
+export type TagVariant = 'info' | 'success' | 'warning' | 'danger' | 'dangerDim' | 'dim';
```

Comment:
rather than 'dim' I think this should be the default style of tag. Should we consider calling ot 'default'?